internal/scheduler/sqlc/query.sql.go

```diff
@@ -1,13 +1,12 @@
-// Code generated by sqlc. DO NOT EDIT.
```

Comment:
Did you manually move and rename package files instead of re-generating using sqlc into a new directory?

Reply:
Btw. You can install `sqlc` and other tools using this make target.
https://github.com/G-Research/armada/blob/master/makefile#L590

Reply:
yeah I did- I'm going to change this again as the next bit of work to move the database to migration scripts so that time I'll do the migration properly.